docs/logs/getting-started-aspnetcore/Program.cs

```diff
@@ -56,9 +56,9 @@
 
 public static partial class ApplicationLogs
 {
-    [LoggerMessage(EventId = 1, Level = LogLevel.Information, Message = "Starting the app...")]
+    [LoggerMessage(Level = LogLevel.Information, Message = "Starting the app...")]
```

Comment:
```suggestion
    [LoggerMessage(LogLevel.Information, "Starting the app...")]
```
I believe you can construct the attribute using positional parameters rather than naming them now. I'm guessing its obvious enough what each parameter is that we can use the more concise syntax.

Reply:
Nice! Let me make the change here.